buona_traduzione.html

```diff
@@ -506,3 +506,7 @@
    </div>
  </body>
 </html>
+<p>
+    <a href="http://validator.w3.org/check?uri=referer"><img
+      src="http://www.w3.org/Icons/valid-xhtml10" alt="Valid XHTML 1.0 Strict" height="31" width="88" /></a>
+  </p>
```

Comment:
Dubito che aggiungendo l'elemento <p> dopo il tag </html> il documento sia ancora valido.

Eviterei in ogni caso di mettere i link al validatore, fanno un po' 1999. :D